mergin/client_pull.py

```diff
@@ -623,7 +627,7 @@ def pull_project_finalize(job):
             raise ClientError("Cannot patch basefile {}! Please try syncing again.".format(basefile))
 
     try:
-        conflicts = job.mp.apply_pull_changes(job.pull_changes, job.temp_dir, job.user_name)
+        conflicts = job.apply_changes()
```

Comment:
I would prefer to keep the direct call to job.mp.apply_pull_changes() instead of introducing a new method in PullJob (it is meant just as a structure to keep everything that is needed for a pull)